setup.py

```diff
@@ -0,0 +1,120 @@
+"""
+Neuraxle Tensorflow Utility classes
+=========================================
+Neuraxle utility classes for tensorflow.
+
+..
+    Copyright 2019, Neuraxio Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+
+from setuptools import setup, find_packages
+
+from neuraxle import __version__ as _VERSION
+
+with open('README.md') as _f:
+    _README = _f.read()
+
+setup(
+    name='neuraxle_tensorflow',
+    version=_VERSION,
+    description='TensorFlow steps, savers, and utilities for Neuraxle. Neuraxle is a Machine Learning (ML) library for building neat pipelines, providing the right '
+                'abstractions to both ease research, development, and deployment of your ML applications.',
+    long_description=_README,
```

Comment:
Please also insert this line here for `.md` readme files to work (otherwise .rst would be assumed by PyPI): 
`    long_description_content_type='text/markdown',`

Reply:
This was not done